JamfSync/Utility/FileShare.swift

```diff
@@ -36,14 +38,21 @@ actor FileShare {
         guard let url = URL(string: "\(type)://\(address)/\(shareName)"), let mountDirectoryUrl = URL(string: "/Volumes") else { throw FileShareMountFailure.mountingFailed }
 
         var mountPoints: Unmanaged<CFArray>?
+        let uiOptions: NSDictionary = [
+            kNAUIOptionKey: kNAUIOptionNoUI
+        ]
         let result = NetFSMountURLSync(url as CFURL,
-                                       mountDirectoryUrl as CFURL,
+                                       nil,
                                        username as CFString?,
                                        password as CFString?,
-                                       nil,
+                                       uiOptions as! CFMutableDictionary?,
```

Comment:
I don't love the ! here, but I couldn't find a way to change it and make it work, and one of the warnings I got when I was messing with it said that the downgrade to CFMutableDictionary can't fail, so I'll call it good, especially since the NSDictionary is statically defined here.